circle.yml

```diff
@@ -6,6 +6,7 @@ machine:
     PARALLEL_TEST_PROCESSORS: 8
 
     PATH: $HOME/.cabal/bin/:$PATH
+    GO_PROJECT: "$HOME/.go_workspace/src/github.com/Originate/gt"
```

Comment:
Thanks for the suggestion! A long-standing mystery resolves. I made it work on my machine, just have to beat Circle into submission. It's 1am here, so I'll do this tomorrow. Created #797 to track this.